h/views/api_groups.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid import security
+from pyramid.httpexceptions import HTTPNoContent, HTTPBadRequest
+
+from h.views.api import api_config
+
+
+@api_config(route_name='api.group_member',
+            request_method='DELETE',
+            link_name='group.member.delete',
```

Comment:
The use of singular vs. plural confused me slightly. The route URL uses plurals ("groups", "members") but the route name uses singular terms ("group", "member") which ends up sounding odd when in the client we convert the route directory into an object where the leaves are methods, eg. `apiClient.group.member.delete(...)`. I suppose it is consistent with the use of `annotation.{create, delete, get, ...}` though.

I suppose it is consistent with the `annotation`